LLama.Examples/UserSettings.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace LLama.Examples;
+
+internal static class UserSettings
+{
+    private static readonly string SettingsFilePath = Path.Join(AppContext.BaseDirectory, "DefaultModel.env");
+
+    private static string? ReadDefaultModelPath()
+    {
+        if (!File.Exists(SettingsFilePath))
+            return null;
+
+        string path = File.ReadAllText(SettingsFilePath).Trim();
+        if (!File.Exists(path))
+            return null;
+
+        return path;
+    }
+
+    private static void WriteDefaultModelPath(string path)
+    {
+        Console.WriteLine($"!!!!WROTE!!!!!!: {SettingsFilePath}");
```

Comment:
Looks like a debug message you intended to remove before pushing?

Reply:
Good catch 😅😅😅